tests/river/unit/analyzer/test_analyzer.py

```diff
@@ -9,231 +9,175 @@
 from river.common.analyzer.sql_join import SqlJoin
 
 
-def test_cache_analysis_redis(patient_mapping):
-    batch_id, resource_id = (uuid4(), uuid4())
+def test_cache_analysis_redis(mimic_mapping):
+    batch_id = uuid4()
+    resource_id = "ckrowsivi00181lpdfod4jpz8"
```

Comment:
we will update mimic_mapping eventually and this will change. I can't really find a satisfying solution (use a fixture with a mapping containing a single resource to avoid hardcoding the id ? using a fake resource_id (eg: not a cuid like `fake-patient-resource-id` ?) but if you can it would be cool. If not, dismiss this comment (meh).